modules/git/commit.go

```diff
@@ -432,16 +431,6 @@ func (c *Commit) GetBranchName() (string, error) {
 	return strings.SplitN(strings.TrimSpace(data), "~", 2)[0], nil
 }
 
-// LoadBranchName load branch name for commit
-func (c *Commit) LoadBranchName() (err error) {
-	if len(c.Branch) != 0 {
-		return
-	}
-
-	c.Branch, err = c.GetBranchName()
-	return err
-}
```

Comment:
Note regarding the changes in this file: I simply noticed randomly that the changes here are unused and hence removed them.